__init__.py

```
@@ -144,20 +166,24 @@ def __init__(self, *args, **kwargs):
+            if not lang2:
+                LOG.warning(f"'{self.root_dir}/locale/{lang}' directory not found! wikipedia will be disabled for '{lang}'")
+                continue
+
+            filename = f"{self.root_dir}/locale/{lang2}/query.intent"
             if not os.path.isfile(filename):
-                LOG.warning(f"{filename} not found! wikipedia common QA will be disabled for '{lang}'")
+                LOG.warning(f"{filename} not found! wikipedia will be disabled for '{lang}'")
                 continue
             samples = []
             with open(filename) as f:
                 for l in f.read().split("\n"):
```

Ruff (0.8.2)

Ambiguous variable name: `l`

(E741)